### PR TITLE
Hot 2186 relationships query cycle time

### PR DESCRIPTION
**Pull request by pokharelmanish**
### Jira Story

- [Relationships query cycle time: Time to add a person to the screening](https://osi-cwds.atlassian.net/browse/HOT-2186)

## Description
Log relationships query cycle time to the new relic.
Note: Please check JIRA for relationships query cycle time screenshot in New relic. 

## Tests
- [x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by pokharelmanish** [comment]
retest this please

File: spec/javascripts/sagas/fetchRelationshipsSagaSpec.js [hunk 1]
```diff
@@ -33,6 +38,18 @@ describe('fetchRelationships', () => {
     expect(gen.next(relationships).value).toEqual(
       put(actions.fetchRelationshipsSuccess(relationships))
     )
+    expect(gen.next().value).toEqual(
+      select(getPersonCreatedAtTimeSelector)
+    )
+    const fetchRelationshipTime = moment().valueOf()
```
Comment (wmitchell-cambria) [inline spec/javascripts/sagas/fetchRelationshipsSagaSpec.js]: I'm surprised that this consistently works. Does it always match up exactly, or is it possible to add a slow delay (like a long for loop) that will cause it to break? Doing something like that will help us make sure that this won't spuriously fail.

Comment (pokharelmanish) [reply spec/javascripts/sagas/fetchRelationshipsSagaSpec.js]: @wmitchell-cambria suggested me another approach which will not break even if there is a long delay. 👍

**Comment by NagaPradeepGajula-osi** [comment]
Seems like the changes are in integration. No more 500 error due to no relationships.

**Comment by pokharelmanish** [comment]
@NagaPradeepGajula-osi cool. Merging now.

**Pull request merged**